sql/updates/world/3.3.5/2018_01_17_01_world.sql

```diff
@@ -0,0 +1,7 @@
+
+-- Reanimated Abomination Abilities
+UPDATE `creature_template` SET `spell1` = 59564, `spell2` = 59576 WHERE `entry` = 31692;
+
```

Comment:
- still missing `<Enter>` on the last line
- remove blank top line
- remove blank line (line 4)

Reply:
This was the hardest part 😂